src/app_desc.rs

```diff
@@ -0,0 +1,46 @@
+
+pub const fn strToCharArray<const C: usize>(s: &str) -> [i8; C] {
```

Comment:
* It should be `str_to_char_array` or better yet: `str_to_cstr_array`. I'm almost sure there is a crate that does this already, but oh well.
* replace `i8` with `c_types::c_char`

Reply:
Will update in the morning.